File: C#/SelectionSort.cs
class SelectionSort{
public static void SelectionSort(int []arr) { 
        int n = arr.Length; 
       // unsorted subarray 
        for (int i = 0; i < n - 1; i++) 
        { 
            // Find the minimum element 
            // in unsorted array 
            int min_idx = i; 
            for (int j = i + 1; j < n; j++) 
                if (arr[j] < arr[min_idx]) 
                    min_idx = j; 
  
            // Swap the found minimum 
            // element with the first 
            // element 
            int temp = arr[min_idx]; 
            arr[min_idx] = arr[i]; 
            arr[i] = temp; 
        }
    } 
}

File: HactoberFest/Java/SelectionSort.java
package HactoberFest.Java;

public class SelectionSort {
    public static void main(String[] args) {
        int[] a={38,52,9,18,6,62,18};
        int min, temp=0;
        for(int i=0; i<a.length; i++){
            min=i;
            for(int j=i+1; j<a.length; j++){
                if(a[j]<a[min]){
                    min=j;
                }
            }
            temp=a[i];
            a[i]=a[min];
            a[min]=temp;
        }
        for(int i=0; i<a.length; i++){
            System.out.print(a[i]+" ");
        }
    }
}
